src/components/ContentSidebar/MetadataSidebar.js

```diff
@@ -157,9 +158,28 @@ class MetadataSidebar extends React.PureComponent<Props, State> {
         const { editors, templates, isLoading, hasError }: State = this.state;
         const showEditor = !!templates && !!editors;
         const showLoadingIndicator = !hasError && !showEditor;
+        const canEdit = this.canEdit();
 
         return (
-            <SidebarContent title={<FormattedMessage {...messages.sidebarMetadataTitle} />}>
+            <SidebarContent
+                title={
+                    <div className='bcs-metadata-header'>
+                        <h3 className='bcs-metadata-title'>
+                            <FormattedMessage {...messages.sidebarMetadataTitle} />
+                        </h3>
+                        {showEditor &&
+                            canEdit && (
+                                <TemplateDropdown
+                                    hasTemplates={templates && templates.length !== 0}
+                                    isDropdownBusy={false}
+                                    onAdd={this.onAdd}
+                                    templates={templates}
+                                    usedTemplates={editors && editors.map((editor) => editor.template)}
```

Comment:
Minor: It looks like this `editors` truthy check and the various default parameter assignments could be replaced by setting `editors: []` in the initialState instance property.

Reply:
Unfortunately don't even need that since I have this guarded with showEditor boolean. But flow hates that.